### PR TITLE
Use the macro GB_TGCLI_IQN_PREFIX instead of the iqn.2016-12.org.glus…

### PR DESCRIPTION
**Pull request by SaravanaStorageNetwork**
…ter-block

Signed-off-by: Saravanakumar Arumugam <sarumuga@redhat.com>
Reviewed-by: Prasanna Kumar Kalever <prasanna.kalever@redhat.com>

### CODE REVIEW

**Comment by SaravanaStorageNetwork** [comment]
@pkalever  PTAL

File: rpc/block_svc_routines.c [hunk 1]
```diff
@@ -3956,8 +3956,8 @@ blockValidateCommandOutput(const char *out, int opt, void *data)
     /* target iqn create validation */
     GB_OUT_VALIDATE_OR_GOTO(out, out, "target iqn creation failed for: %s",
                     cblk, cblk->volume,
-                    "Created target iqn.2016-12.org.gluster-block:%s.",
-                    cblk->gbid);
+                    "Created target  %s:%s.",
```
Comment (pkalever) [inline rpc/block_svc_routines.c]: Very minor correction.
Can you please merge line 3959 at 3958

Comment (SaravanaStorageNetwork) [reply rpc/block_svc_routines.c]: Done. @pkalever  PTAL

File: rpc/block_svc_routines.c [hunk 2]
```diff
@@ -3955,9 +3955,8 @@ blockValidateCommandOutput(const char *out, int opt, void *data)
 
     /* target iqn create validation */
     GB_OUT_VALIDATE_OR_GOTO(out, out, "target iqn creation failed for: %s",
-                    cblk, cblk->volume,
-                    "Created target iqn.2016-12.org.gluster-block:%s.",
-                    cblk->gbid);
+                    cblk, cblk->volume, "Created target  %s:%s.",
```
Comment (pkalever) [inline rpc/block_svc_routines.c]: You seem to have kept an extra space after "Created target", Can you please remove this addition.
Thanks!

**Comment by pkalever** [comment]
Looks good to me, squash them please

**Comment by pkalever** [comment]
Thanks @SaravanaStorageNetwork :-)

**Pull request merged**